src/test/lambda/local/util.ts

```diff
@@ -26,11 +26,7 @@ export async function createWorkspaceFolder(prefix: string): Promise<{
     }
 }
 
-export async function createTemporaryDirectory(prefix: string): Promise<string> {
-    const fullPrefix = path.join(os.tmpdir(), prefix)
-
-    return await filesystem.mkdtempAsync(fullPrefix)
-}
+export const  createTemporaryDirectory = filesystem.mkdtempAsync
```

Comment:
two spaces between const and createTempDir

Reply:
✔